docs/adoption.md

```diff
@@ -130,6 +130,36 @@ Once this is done, we will need to set the `AZUREAPPSERVICE_PUBLISHPROFILE_...`
 
 Finally, in GitHub actions secrets, set `AZUREAPPSERVICE_PUBLISHPROFILE_PRODUCTION` and `AZUREAPPSERVICE_PUBLISHPROFILE_STAGING` to the contents of the respective publish profiles downloaded previously. Once this is complete, you can rerun the workflow and all of the steps should pass.
 
+### Deploy housing-repairs-online-api
```

Comment:
Given `housing-repairs-online-api` depends on this API being deployed, I suggest moving this section above the section for `housing-repairs-online-api`.